handlers.go

```diff
@@ -190,6 +190,11 @@ func (s Server) resourcePatchHandler(w http.ResponseWriter, r *http.Request, id
 		return
 	}
 
+	if resource.Attributes == nil {
```

Comment:
One more thing! Can you replace this line into the following?
```go
if len(resource.Attributes) == 0 {
```
This way we check for empty array and nil values.

Reply:
fixed it!https://github.com/elimity-com/scim/pull/102/commits/fa9d5e7d34ed030d2a58a1eea0e66d6843fc8875